### PR TITLE
Updated main docs page to have more information

### PR DESCRIPTION
**Pull request by jhaack**
#896 More informative base page. Also moved deprecated examples.

---
This change is [Reviewable](https://reviewable.io/reviews/volttron/volttron/903)

### CODE REVIEW

File: docs/source/README.rst [hunk 1]
```diff
@@ -1,27 +1,50 @@
...
+    volttron -vv -l volttron.log&
+
+This enters the virtual Python environment and then starts the platform in debug (vv) mode with a log file named
+volttron.log.
+
+Next, start an example listener to see it publish and subscribe to the message bus:
+
+    scripts/core/make-listener
+
+This script handles several different commands for installing and starting an agent after removing an old copy. This
```
Comment (craig8) [inline docs/source/README.rst]: I don't think this rendering is what you had in mind.
index-error-rendering

File: docs/source/README.rst [hunk 2]
```diff
@@ -33,9 +56,10 @@ Support
 -------
 
 If you are having issues, please let us know.
```
Comment (craig8) [inline docs/source/README.rst]: STACKOVERFLOW!!!!!

File: docs/source/specifications/configuration-store.rst [hunk 3]
```diff
@@ -1,3 +1,5 @@
+.. _ConfigurationStore:
+
 Agent Configuration Store
```
Comment (craig8) [inline docs/source/specifications/configuration-store.rst]: Seems like hte other pages have a
============================================
Title
============================================

Perhaps we should keep that constistancy or change to the other way?

Comment (craig8) [inline docs/source/specifications/configuration-store.rst]: hmm you need to look at the edit of that to see the difference ha

Comment (jhaack) [inline docs/source/specifications/configuration-store.rst]: Going to have an editor go through and make pages consistent.

**Pull request merged**